coordinatorLogin/coordinatorLogin/Coordinators/MainCoordinator.swift

```diff
@@ -0,0 +1,83 @@
+//  MainCoordinator.swift
+//  coordinatorLogin
+//  Created by angelina on 28.12.2022.
+
+import UIKit
+
+/// Главный координатор
+final class MainCoordinator: NSObject, Coordinator, UINavigationControllerDelegate {
+    
+    // MARK: - Constants
+    private enum Constants {
+        static let keyString = "isLogin"
+        static let loginString = "q"
+    }
+    
+    // MARK: - Public property
+    var childCoordinators = [Coordinator]()
```

Comment:
```suggestion
    var childCoordinators = [Coordinator]()
```
это свойство точно нельзя сделать приватным?)

Reply:
низя, ругается